test/parallel/test-cli-node-options.js

```diff
@@ -53,33 +23,25 @@ expect('--track-heap-objects', 'B\n');
 expect('--throw-deprecation', 'B\n');
 expect('--zero-fill-buffers', 'B\n');
 expect('--v8-pool-size=10', 'B\n');
+
 if (common.hasCrypto) {
   expect('--use-openssl-ca', 'B\n');
   expect('--use-bundled-ca', 'B\n');
   expect('--openssl-config=_ossl_cfg', 'B\n');
 }
 
 // V8 options
-expect('--abort-on-uncaught-exception', 'B\n');
-expect('--abort_on_uncaught_exception', 'B\n');
 expect('--abort_on-uncaught_exception', 'B\n');
-expect('--max_old_space_size=0', 'B\n');
-expect('--max-old_space-size=0', 'B\n');
 expect('--max-old-space-size=0', 'B\n');
 
 function expect(opt, want) {
-  const printB = require.resolve('../fixtures/printB.js');
-  const argv = [printB];
-  const opts = {
-    env: {NODE_OPTIONS: opt},
-    maxBuffer: 1000000000,
-  };
-  exec(process.execPath, argv, opts, common.mustCall(function(err, stdout) {
+  const argv = ['-e', 'console.log("B")'];
```

Comment:
If all we're doing with `-e` is `console.log()` perhaps consider using `-p` instead?